lang/cs/Org.Apache.REEF.Common.Tests/metrics/MetricTestUtils.cs

```diff
@@ -175,5 +175,32 @@ public IMetricsRecordBuilder CreateRecord(IMetricsInfo info)
                 throw new System.NotImplementedException();
             }
         }
+
+        /// <summary>
+        /// Metrics visitor implementation.
+        /// </summary>
+        internal sealed class MetricsVisitorForTests : IMetricsVisitor
+        {
+            public long CounterValue { get; private set; }
+
+            public long LongGauge { get; private set; }
+
+            public double DoubleGauge { get; private set; }
+
+            public void Gauge(IMetricsInfo info, long value)
+            {
+                LongGauge = value;
+            }
+
+            public void Gauge(IMetricsInfo info, double value)
```

Comment:
As this is the only implementation of IMetricsVisitor, you might want to show how IMetricsInfo is used in the method as an example.